test/test.venue.txt

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
What is the reason for this file?

Reply:
Since venues model in it current form is so big i made this file for testing purposes. I figured you might want to test things too so i left it as a template.